docs/SPIR-V.rst

```diff
@@ -231,19 +231,24 @@ Textures
 `Texture types <https://msdn.microsoft.com/en-us/library/windows/desktop/bb509700(v=vs.85).aspx>`_
 are translated into SPIR-V ``OpTypeImage``, with parameters:
 
-====================   ========== ===== ======= == ======= ================ =================
-HLSL Texture Type          Dim    Depth Arrayed MS Sampled  Image Format       Capability
-====================   ========== ===== ======= == ======= ================ =================
-``Texture1D``          ``1D``      0       0    0    1     ``Unknown``
-``Texture2D``          ``2D``      0       0    0    1     ``Unknown``
-``Texture3D``          ``3D``      0       0    0    1     ``Unknown``
-``TextureCube``        ``Cube``    0       0    0    1     ``Unknown``
-``Texture1DArray``     ``1D``      0       1    0    1     ``Unknown``
-``Texture2DArray``     ``2D``      0       1    0    1     ``Unknown``
-``TextureCubeArray``   ``3D``      0       1    0    1     ``Unknown``
-``Buffer<T>``          ``Buffer``  0       0    0    1     Depends on ``T`` ``SampledBuffer``
-``RWBuffer<T>``        ``Buffer``  0       0    0    2     Depends on ``T`` ``SampledBuffer``
-====================   ========== ===== ======= == ======= ================ =================
+====================    ========== ===== ======= == ======= ================ =================
+HLSL Texture Type           Dim    Depth Arrayed MS Sampled  Image Format       Capability
+====================    ========== ===== ======= == ======= ================ =================
+``Texture1D``           ``1D``      0       0    0    1     ``Unknown``
+``Texture2D``           ``2D``      0       0    0    1     ``Unknown``
+``Texture3D``           ``3D``      0       0    0    1     ``Unknown``
+``TextureCube``         ``Cube``    0       0    0    1     ``Unknown``
+``Texture1DArray``      ``1D``      0       1    0    1     ``Unknown``
+``Texture2DArray``      ``2D``      0       1    0    1     ``Unknown``
+``TextureCubeArray``    ``3D``      0       1    0    1     ``Unknown``
+``Buffer<T>``           ``Buffer``  0       0    0    1     Depends on ``T`` ``SampledBuffer``
+``RWBuffer<T>``         ``Buffer``  0       0    0    2     Depends on ``T`` ``SampledBuffer``
+``RWTexture1D<T>``      ``1D``      0       0    0    2     Depends on ``T``
```

Comment:
Do we need extra capability here? I think it should also require `SampledBuffer`?

Reply:
No, SampledBuffer capability is only required when the Dimension parameter is "Buffer".
In the RWTexture... cases, the dimension is 1D, 2D, or 3D.

Reply:
I see. Thanks!